doc/reference/power_management/index.rst

```diff
@@ -339,7 +339,7 @@ Device Set Power State
 
 .. code-block:: c
 
-   int device_set_power_state(struct device *device, u32_t device_power_state);
+   int device_set_power_state(struct device *device, u32_t device_power_state, device_pm_cb cb);
```

Comment:
How I can pass custom data to the callback?
(We agreed on a slightly different API: https://github.com/zephyrproject-rtos/zephyr/pull/13382#discussion_r256769700)

Reply:
"int (*device_pm_control)(struct device *device, u32_t command, void *arg, device_pm_cb cb, void *context)
typedef void (*device_pm_cb)(struct device *dev, int status, void *context);"

I missed the arg parameter you mentioned in the comment.
How are planning to use arg parameter?

Reply:
The arg allows to pass some parameters specific for given operation. I not have exact scenario for its usage now, however many times I missed this functionality.

Reply:
OK sure, will update the PR accordingly.